src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -1386,7 +1380,8 @@ static void window_editor_scenario_options_park_invalidate(rct_window* w)
  */
 static void window_editor_scenario_options_park_paint(rct_window* w, rct_drawpixelinfo* dpi)
 {
-    int32_t x, y, arg;
+    int32_t arg;
+    ScreenCoordsXY screenCoords;
```

Comment:
```suggestion
    ScreenCoordsXY screenCoords{};
```
Initialize it, just to be sure